content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,197 @@
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience with the streaming of logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs) as well. One or more executors are available by default on Cloud instances of Sourcegraph.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Batch_Changes_Side_by_Side',
+    mp4: 'blog/release-post/4.0/Batch_Changes_Side_by_Side'
+  }}
+  loop={true}
+  title="Server-side Batch Changes"
+  caption="Side by side comparison of deploying Batch Changes locally vs. server-side"
+/>
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+
+### ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Get started with Sourcegraph faster than ever
+After many years of building Sourcegraph into a secure and highly-scalable solution, we're ready to bring it to the cloud. Sourcegraph Cloud now offers dedicated, single-tenant instances of Sourcegraph, making it easier than ever for new teams to get started. You can read more in our announcement blog here[/blog/enterprise-cloud].
+
+[Docs](https://docs.sourcegraph.com/cloud)
+
+### 💖 Advanced admin capabilities
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
+Multi-version upgrades help to reduce the burden and time required to upgrade to the latest version of Sourcegraph. Customers will now be able to seamlessly upgrade from any 3.20+ version to Sourcegraph 4.0 in just a few simple steps. With faster upgrades, customers can quickly start using the new features and enhancements that are released every month. 
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Multi-version%20Upgrades',
+    mp4: 'blog/release-post/4.0/Multi-version%20Upgrades'
+  }}
+  loop={true}
+  title="Multi-version upgrades"
+  caption="Demo of multi-version upgrades"
+/>
+
+[Docs](https://docs.sourcegraph.com/admin/updates#multi-version-upgrades)
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### View usage and measure the value of our platform with new and enhanced in-product analytics
```

Comment:
@lcolston08 - should we lead with this in the advanced admin capabilities section?

Reply:
I moved it around so that we lead with multi-version updates since it seems like this post is most relevant to existing customers and we've already introduced in-product analytics with a recent blog post.